dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java

```diff
@@ -759,9 +759,13 @@ else if( fieldbcontentname.startsWith("date") ){
 		    }
 		}
 
-		if (!UtilMethods.isSet(orderBy)){
-			orderBy = "modDate desc";
-		}
+        if (!UtilMethods.isSet(orderBy)){
+            orderBy = "modDate desc";
+        }else{
+            if(orderBy.charAt(0)=='.'){
+                orderBy = st.getVelocityVarName() + orderBy;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "orderBy". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1226)